backend/mora/service/orgunit.py

```diff
@@ -121,7 +121,7 @@ def prepare_create(self, req):
         )
 
         unitid = util.get_uuid(req, required=False) or str(uuid.uuid4())
-        bvn = util.checked_get(req, mapping.USER_KEY, unitid)
+        bvn = util.checked_get(req, mapping.USER_KEY, unitid) or unitid
```

Comment:
Passing `null` or `None` here should trigger the fallback if needed; disregarding `""` — i.e. the empty string — is unusual and inconsistent with our other APIs. (See also https://github.com/os2mo/os2mo/commit/bc54c1bc6e28.)

Reply:
Fedt - den er jo helt for sig selv. Kan man fjerne et enkelt commit fra pullrequestet?

Reply:
Prøv `git rebase -i`?

Reply:
Nu vil det bryde med løsningen at acceptere den tomme streng, meningen er jo netop at man lader brugeren vælge om han vil angive den eller ikke. Ellers skal man jo lave noget frontend, der gør at man ikke overfører den, hvis den ikke er udfyldt - er det ikke lidt bagvendt med model etc?

Reply:
Done - null is now enforced when not filled